app/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixListData.kt

```diff
@@ -7,8 +7,8 @@
 
 package mozilla.components.lib.publicsuffixlist
 
-import mozilla.components.lib.publicsuffixlist.ext.binarySearch
```

Comment:
Can we exclude the mozilla dir from reformats? This might make future updates more difficult if their commits don't apply cleanly anymore.

Reply:
Don't know if it's doable, reverted the hunk for now.